saleor/graphql/app/utils.py

```diff
@@ -0,0 +1,23 @@
+from django.core.exceptions import ValidationError
+
+from ..account.utils import get_out_of_scope_permissions
+from ..core.enums import AppErrorCode
+from ..utils import requestor_is_superuser
+
+
+def ensure_can_manage_permissions(requestor, permission_items):
```

Comment:
We have the same functionality in the few more places. Maybe we should do this more generic, returning just ValidationError(error_msg, code=code, params=params) and handling codename in mutations (or we can pass codename as an argument).

Reply:
Yes, I was thinking about this, but it needs to be done as a separate PR as this branch will be merged into the feature branch. Will create a gh issue for that